rust/unique_string_demo.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
```

Comment:
The compilation process is a bit convoluted for `unique_string_demo.rs`, so use this sh script to aid the process.